import.js

```diff
@@ -33,37 +33,35 @@ const importFile = (octokit, file, values) => {
         }
         const createPromises = csvRows.map((row) => {
           const sendObj = {
-            owner: values.userOrOrganization,
-            repo: values.repo,
-            title: row[titleIndex],
+            issue : {}
           };
 
+          sendObj.issue.title = row[titleIndex];
+
           // if we have a body column, pass that.
-          if (bodyIndex > -1) {
-            sendObj.body = row[bodyIndex];
+          if (bodyIndex > -1 && row[bodyIndex] !== "") {
+            sendObj.issue.body = row[bodyIndex];
           }
 
           // if we have a labels column, pass that.
           if (labelsIndex > -1 && row[labelsIndex] !== "") {
-            sendObj.labels = row[labelsIndex].split(",");
+            sendObj.issue.labels = row[labelsIndex].split(",");
           }
 
           // if we have a milestone column, pass that.
           if (milestoneIndex > -1 && row[milestoneIndex] !== "") {
-            sendObj.milestone = row[milestoneIndex];
+            sendObj.issue.milestone = row[milestoneIndex];
           }
 
           // if we have an assignee column, pass that.
           if (assigneeIndex > -1 && row[assigneeIndex] !== "") {
-            sendObj.assignees = row[assigneeIndex].replace(/ /g, "").split(",");
+            sendObj.issue.assignee = row[assigneeIndex].replace(/ /g, "").split(",");
```

Comment:
This does not work. Try importing `https://github.com/gavinr/github-csv-tools/blob/master/test/4.csv` (change the username in the csv to your username). I get error:

```
message: 'Invalid request.\n' +
        '\n' +
        `For 'properties/assignee', ["MyUserName"] is not a string or null.`,
```

Reply:
I fixed the assignee problem and I test with all tests files.
All are OK (noTitle.csv return "Title required by GitHub, but not found in CSV.")